### ISSUE TITLE
Event log module support

### ISSUE THREAD

**Issue by AndreMiras** [corpetty/py-etherscan-api#26]
py-etherscan-api currently provide event logs support. That would be a nice one to have. See API documentation here: https://etherscan.io/apis#logs

**Comment by corpetty** [comment]
I agree, this would be great. I'll review the pull requests and add soon.  Thanks for contributing!

**Comment by AndreMiras** [comment]
Yes I can work on it, pretty soon. I've a working version locally for a different project.
Do you mind if we tackle https://github.com/corpetty/py-etherscan-api/issues/20 a bit before growing the code base. I can also help with this. I would like to at least have some tox testing plus continuous integration like Travis. For the CI I usually build it within a Dockerfile so I can really isolate and specify the whole environment. But if you think it's too much I can go the old fashion way. The annoying thins is Travis is running old OS. With Docker you put the (Linux) OS you want.

**Comment by corpetty** [comment]
20 is closed.

**Comment by AndreMiras** [comment]
Great thanks! I'll tackle the little missing tasks first because I'm too busy for the big one. But I keep it for later :)
